### PR TITLE
Update [g-m]* from monorail results

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See #798 for methodology

### CODE REVIEW

File: media-source/dedicated-worker/META.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/878133
```
Comment (stephenmcgruer) [inline media-source/dedicated-worker/META.yml:3]: Generic implementation tracking bug

File: mediacapture-record/META.yml [hunk 2]
```diff
@@ -8,3 +8,8 @@ links:
           subtest: MediaRecorder will fire an exception when stopped after having just been stopped
         - test: MediaRecorder-stop.html
           subtest: MediaRecorder will fire an exception when stopped after having just been spontaneously stopped
+    - product: chrome
+      url: https://crbug.com/884059
```
Comment (stephenmcgruer) [inline mediacapture-record/META.yml:12]: Appears unrelated

File: mediacapture-streams/META.yml [hunk 3]
```diff
@@ -46,3 +46,20 @@ links:
...
+    - product: chrome
+      url: https://crbug.com/1149397
+      results:
+        - test: idlharness.https.window.html
+    - product: chrome
+      url: https://crbug.com/1149999
+      results:
+        - test: MediaDevices-after-discard.https.html
+    - product: chrome
+      url: https://crbug.com/770308
```
Comment (stephenmcgruer) [inline mediacapture-streams/META.yml:58]: Seems unrelated.

File: mediacapture-streams/META.yml [hunk 4]
```diff
@@ -46,3 +46,20 @@ links:
...
+      results:
+        - test: MediaDevices-enumerateDevices.https.html
+        - test: MediaStream-MediaElement-srcObject.https.html
+    - product: chrome
+      url: https://crbug.com/1099841
```
Comment (stephenmcgruer) [inline mediacapture-streams/META.yml:63]: Unrelated

File: mst-content-hint/META.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/884059
```
Comment (stephenmcgruer) [inline mst-content-hint/META.yml:3]: Seems unrelated.

File: mathml/relations/html5-tree/META.yml [hunk 6]
```diff
@@ -10,3 +10,7 @@ links:
           status: FAIL
         - test: integration-point-4.html
           status: FAIL
+    - product: chrome
+      url: https://crbug.com/1093840
```
[the remark this answers is not in the document]
Comment (foolip) [reply mathml/relations/html5-tree/META.yml:14]: Sure, even if turns out it's not the only cause of failure, it's still relevant.

File: media-source/META.yml [hunk 7]
```diff
@@ -102,3 +102,15 @@ links:
           subtest: 'MediaSource interface: attribute onsourceclose'
         - test: idlharness.window.html
           subtest: 'MediaSource interface: mediaSource must inherit property "onsourceclose" with the proper type'
+    - product: chrome
+      url: https://crbug.com/641121
```
[the remark this answers is not in the document]
Comment (foolip) [reply media-source/META.yml:106]: Looks likely relevant, the test failure is about event order and issue 641121 is about state change order.

File: intersection-observer/META.yml [hunk 8]
```diff
@@ -11,3 +11,7 @@ links:
           subtest: IntersectionObserverEntry interface object length
         - test: idlharness.window.html
           subtest: 'IntersectionObserver interface: observer must inherit property "thresholds" with the proper type'
+    - product: chrome
+      url: https://crbug.com/870947
```
Comment (foolip) [inline intersection-observer/META.yml:15]: Confirmed this one. This bug could be turned into something more descriptive since there's only a single subtest failure, but it's the right bug.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@stephenmcgruer there are test failures so I'll leave this to you.
